client/client_test.go

```diff
@@ -477,7 +477,7 @@ func testExportedImageLabels(t *testing.T, sb integration.Sandbox) {
 
 	hasLabel := func(dgst digest.Digest) bool {
 		for k, v := range info.Labels {
-			if strings.HasPrefix(k, "containerd.io/gc.ref.content.") && v == dgst.String() {
```

Comment:
This change is causing the tests to fail. `containerd.io` is the correct name, I'm guessing the rename accidentally hit this.

Reply:
Ooops. Accidental mass regex false positive, yep.

Reply:
Fixed